docs/marionette.view.md

```diff
@@ -482,6 +484,63 @@ In this example, the doubly-nested view structure will be rendered in a single p
 This system is recursive, so it works for any deeply nested structure. The child views
 you show can render their own child views within their onBeforeShow callbacks!
 
+### Listening to events on children
+
+Using regions lets you listen to the events that fire on child views - views
+attached inside a region. This lets a parent view take action depending on what
+is happening in views it directly owns.
+
+**To see more information about events, see the [events](#events) section**
+
+#### Defining `childViewEvents`
+
+The `childViewEvents` hash defines the events to listen to on a view's children
+mapped to the method to call. The method will receive a `child` object
+referencing the view that triggered the event.
+
+```javascript
+var Mn = require('backbone.marionette');
+
+let MyView = Mn.View.extend({
+  regions: {
+    product: '.product-hook'
+  },
+
+  childViewEvents: {
+    'add:item': 'updateBasketValue'
+  },
+
+  /** Assume item is the model belonging to the child */
+  updateBasketValue: function(child, item) {
+    var initialValue = this.model.get('value');
+    this.model.set({
+      value: item.get('value') * item.get('quantity')
+    });
+  }
+});
+```
+
+You can also directly define a function to call in `childViewEvents` like so:
+
+```javascript
+var Mn = require('backbone.marionette');
+
+let MyView = Mn.View.extend({
+  regions: {
+    product: '.product-hook'
+  },
+
+  childViewEvents: {
+    'add:item': function(child, item) {
```

Comment:
Probably.  If we es6 things we should be consistent.  Plus this is a case for `const` anyhow